enterprise-suite/es-grafana/akka-exporter.json

```diff
@@ -1,13 +1,13 @@
 [  
   {"graphName":"Mailbox Size", "promQL":["max by (actor)(akka_actor_mailbox_size{workload=\"$workloadId\"})"]},
   {"graphName":"Mailbox Time", "promQL":["max by (actor)(akka_actor_mailbox_time_ns{workload=\"$workloadId\",quantile=\"0.99\"})"]},
-  {"graphName":"Actor Failures", "promQL":["sum by (actor) (akka_actor_actor_failure{workload=\"$workloadId\"})"]},
-  {"graphName":"Processed Messages", "promQL":["sum by (actor)(irate(akka_actor_processed_messages{workload=\"$workloadId\"}[1m]))"]},      
+  {"graphName":"Actor Failures per 15 Minutes", "promQL":["sum by (actor) (rate(akka_actor_actor_failure{workload=\"$workloadId\"}[15m]))"]},
+  {"graphName":"Processed Message Rate", "promQL":["sum by (actor)(irate(akka_actor_processed_messages{workload=\"$workloadId\"}[1m]))"]},      
   {"graphName":"Processing Time", "promQL":["max by (actor)(akka_actor_processing_time_ns{workload=\"$workloadId\",quantile=\"0.99\"})"]},
-  {"graphName":"Sent Messages", "promQL":["sum by (actor)(irate(akka_actor_sent_messages{workload=\"$workloadId\"}[1m]))"]},
+  {"graphName":"Sent Messages Rate", "promQL":["sum by (actor)(irate(akka_actor_sent_messages{workload=\"$workloadId\"}[1m]))"]},
   {"graphName":"Running Actors", "promQL":["sum by (actor)(akka_actor_running_actors{workload=\"$workloadId\"})"]},
-  {"graphName":"Unhandled Messages", "promQL":["sum by (actor)(akka_actor_unhandled_message{workload=\"$workloadId\"})"]},
-  {"graphName":"Dead Letters", "promQL":["sum by (actor) (akka_actor_dead_letter{workload=\"$workloadId\"})"]},
+  {"graphName":"Unhandled Messages Rate", "promQL":["sum by (actor)(irate(akka_actor_unhandled_message{workload=\"$workloadId\"}[1m]))"]},
+  {"graphName":"Dead Letters per Minute", "promQL":["sum by (actor) (irate(akka_actor_dead_letter{workload=\"$workloadId\"}[1m]))"]},
```

Comment:
"per Minute" should be changed to  "Rate" or "per Second".    "Rate" looks like the consistent thing to say here